src/Console/InstallCommand.php

```diff
@@ -269,16 +269,15 @@ protected function installInertiaStack()
         $this->updateNodePackages(function ($packages) {
             return [
                 '@inertiajs/inertia' => '^0.8.2',
```

Comment:
Might be worth bumping this to `0.8.4`, given the recent axios vulnerability (see [here](https://github.com/inertiajs/inertia/releases/tag/v0.8.4)).
```suggestion
                '@inertiajs/inertia' => '^0.8.4',
```